CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## Unreleased
+
 ## [3.3.1+portage-3.2.0] - 2023-07-26
```

Comment:
You need to remove this line 

```
## [3.3.1+portage-3.2.0] - 2023-07-26
```
with the release version and date as we have not committed to the release yet.

Reply:
updated